app/controllers/users_controller.rb

```diff
@@ -1,10 +1,19 @@
 class UsersController < ApplicationController
   layout "main_layout"
 
-  def index; end
+  def index
+    @user = current_user
+    @gds = current_user.permissions.team_management
+    if @gds
+      @teams = Team.all
+    else
+      @team = current_user.team
+    end
+  end
 
   def invite
     @form = InviteUserForm.new({})
+    @team = params[:team_id] if params[:team_id].present?
```

Comment:
this is a bit redundant? the `if` statement

Reply:
This has been removed